Xabe.FFmpeg/FFmpeg.cs

```diff
@@ -56,10 +58,20 @@ public abstract class FFmpeg
 
         /// <summary>
         ///     Download latest FFmpeg version for current operating system to FFmpeg.ExecutablePath. If it is not set download to ".".
+        /// <param id="gpuSupport">Determine if a fully compiled FFPmpeg is downloaded</param>
         /// </summary>
-        public static Task GetLatestVersion()
+        public static Task GetLatestVersion(bool gpuSupport = false)
```

Comment:
Probably GPU support is not only one feature in full version. Maybe this param should be named as 'fullVersion'. Description about differences in this two versions should be in comment.